docs/usage/modularizing.md

```diff
@@ -0,0 +1,77 @@
+
+(modularizing-python-codes)=
+# Modularizing Python codes
+
+Pyodide provides a simple API {any}`pyodide.runPython` to run Python codes.
+However, when your Python codes grow bigger, putting hundreds of lines inside `runPython` is not scalable.
+
+The best way to run Python code with Pyodide is:
+
+1. write a Python package
+1. load your Python package into the Pyodide (Emscripten) virtual file system
+1. import the package with ``let mypkg = pyodide.pyimport("mypkgname")``
+1. call into your package with ``mypkg.some_api(some_args)``.
+
+## How to load Python package into the virtual file system
```

Comment:
I added a simple example of this here:
https://github.com/pyodide/pyodide-examples/
I think it'd be good to encourage people to use a dev server that zips stuff on the fly like what I put there:
https://github.com/pyodide/pyodide-examples/blob/main/server.py

Reply:
One step further, I think it would be nice if we provide a way to integrate the development process with `webpack devserver` of `rollup devserver`